src/index.html

```diff
@@ -4,19 +4,108 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>MOYO header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. If this is meant to link to the home page, use `href="/"` or `href="./"` instead of `href="index.html"`. Anchor links should start with the `#` symbol if they are meant to link to an anchor on the page.